### PR TITLE
US125733/add discussion attachments list ui

### PR DESCRIPTION
**Pull request by mbosiljevac**
Double checked with Fiona and confirmed design
This PR gets the attachmentsList data for a discussion post and displays it.
Todo: Hookup action to download attachments.
image

### CODE REVIEW

File: components/left-panel/discussions/consistent-evaluation-discussion-post-page.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import './consistent-evaluation-discussion-evidence-body';
+import { attachmentClassName, attachmentListClassName, threadRel } from '../../controllers/constants.js';
 import { css, html, LitElement } from 'lit-element';
 import { Classes } from 'd2l-hypermedia-constants';
 import { LocalizeConsistentEvaluation } from '../../../localize-consistent-evaluation.js';
 import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
 import { SkeletonMixin } from '@brightspace-ui/core/components/skeleton/skeleton-mixin.js';
-import { threadRel } from '../../controllers/constants.js';
```
Comment (jeroach) [inline components/left-panel/discussions/consistent-evaluation-discussion-post-page.js:7]: Was `threadRel` just not being used? I don't see any usages removed in the file

Comment (mbosiljevac) [reply components/left-panel/discussions/consistent-evaluation-discussion-post-page.js:7]: the import just got moved to line 2!

Comment (jeroach) [reply components/left-panel/discussions/consistent-evaluation-discussion-post-page.js:7]: ah, my bad - missed that :)

**Review by jeroach — APPROVED** [review]
LGTM

**Review by JamieTran — APPROVED** [review]

**Pull request merged**